include/raft.h

```diff
@@ -508,12 +508,13 @@ enum { RAFT_UNAVAILABLE, RAFT_FOLLOWER, RAFT_CANDIDATE, RAFT_LEADER };
  */
 struct raft_progress
```

Comment:
Just confirming (sorry, not familiar with code), but this struct is in-memory only and never persisted to disk or exchange over the network, right?

Always triggered by addition of struct members in the middle of a struct ;)

Reply:
Yes, it's in-memory only.